### DISCUSSION TITLE
Welcome to discussions Discussions!

### DISCUSSION THREAD

**Discussion by VoyagerPWS** [das-developers/discussions#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

  To start new discussions, use the green [new discussions](https://github.com/das-developers/discussions/new) button at the top of [Discussions](https://github.com/das-developers/discussions).

**Comment by VoyagerPWS** [comment]
Oh, by the way, the above is mostly the stock template.

**Comment by cpiker** [comment]
I think the initial comment should point out how to start new threads, maybe something like this for the last line:
```
To start a new topic/discussion, please use this link: https://github.com/das-developers/discussions/new.
```
Otherwise we just end up with all post under the same thread.  Could you edit the initial post?

**Reply by VoyagerPWS** [reply]
Done
